addons/pinniped/config-controller/controllers/controllers.go

```diff
@@ -75,6 +75,10 @@ func (c *PinnipedController) SetupWithManager(manager ctrl.Manager) error {
 	return nil
 }
 
+// +kubebuilder:rbac:resources=configmaps,verbs=list;watch;get;patch;update;delete
```

Comment:
I'm good with them being at the top of `Reconcile` as this seems to be where others put these annotations and currently this is where we are at least making initial requests.

Reply:
seems good to me as well!

Reply:
I think we want to `list;watch;get;patch;update;delete` secrets as well as configmaps (looks like we just have the `list;watch;get` verbs for secrets right now)

Reply:
I think we want to `list;watch;get;patch;update;delete`, but for configmaps we only need to `list;watch;get`

Reply:
swapped!

Reply:
I think we need the `list` in there for how informers work, but I could be wrong. I'll check that also.